docs/rules/no-memoize-on-static.md

```
@@ -3,3 +3,65 @@
+
+class ConfigLoader {
+  constructor(private readonly tenant: string) {}
+
+  @Memoize()
+  load() {
+    return fetchConfigFor(this.tenant); // cache scoped to the instance/request
+  }
+}
+```
+
+### ✅ Good
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)